samples/mailjet.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// sample-metadata:
+//  title: Mailjet
+//  usage: node mailjet
+
+'use strict';
+
+// [START send]
```

Comment:
Can we use proper product naming for the tag please. Thanks

Reply:
its just copy of current files. should i change them to new one?
https://github.com/googleapis/nodejs-compute/blob/master/samples/mailjet.js

Reply:
Feel free to correct the old files as well. New files have to pass the license header and tag checks, otherwise we can't merge.